README.md

```diff
@@ -336,10 +409,10 @@ const resolvedTasks = tasksCollection.filter({ resolved: true })
 resolvedTasks.length // => 3
 ```
 
-#### `find(query: Object): ?Model`
+#### `find(query: Object, { required?: boolean = false }): ?Model`
 
 Same as `filter` but it will halt and return when the first model matches
-the conditions.
+the conditions. If `required` it will raise an error if not found.
```

Comment:
An example of the error thrown would be useful here.

Reply:
done